Documentation/ApiOverview/SessionStorageFramework/Index.rst

```diff
@@ -6,10 +6,44 @@ Session Storage Framework
 =========================
 
 As of version 8.6, TYPO3 comes with the option to choose between different storages for both frontend end backend user
-sessions. Previously, all sessions were stored in the database in the tables `fe_sessions`, `fe_session_data` and
-`be_sessions`.
+sessions (called session backends). Previously, all sessions were stored in the database in the tables `fe_sessions`, `fe_session_data` and
+`be_sessions` respectively.
+
+The Core ships two session backends by default:
+- Database storage
+- Redis storage
+
+By default user sessions are still stored in the database using the database storage backend, but the former table `fe_session_data` is now obsolete and therefore removed.
```

Comment:
sry. wrong repo. should have gone against official one. will do the PR again there and will fix that.